index.bs

```diff
@@ -1078,17 +1079,26 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     The [=credential private key=] is bound to a particular [=authenticator=] - its [=managing authenticator=] -
     and is expected to never be exposed to any other party, not even to the owner of the [=authenticator=].
 
-    Note that in the case of [=self
-    attestation=], the [=credential key pair=] is also used as the [=attestation key pair=], see [=self attestation=]
-    for details.
+    Note: In the case of [=self attestation=], the [=credential key pair=] is also used as the [=attestation key pair=], see [=self attestation=] for details.
 
     Note: The [=credential public key=] is referred to as the [=user public key=] in FIDO UAF [[UAFProtocol]], and in FIDO U2F
-    [[FIDO-U2F-Message-Formats]] and some parts of this specification that relate to it.
+    [[FIDO-U2F-Message-Formats]] and some parts of this specification that relate to it. Also, the term [=user credential=] is occasionally used to synonymously refer to [=credential public key=] and [=user public key=].
 
 : <dfn>Credential Properties</dfn>
 :: A [=credential property=] is some characteristic property of a [=public key credential source=], such as whether it is a
     [=client-side discoverable credential=] or a [=server-side credential=].
 
+: <dfn>Hardware-bound Device Key Pair</dfn>
+: <dfn>Device-bound Key</dfn>
+: <dfn>Device Private Key</dfn>
+: <dfn>Device Public Key</dfn>
+: <dfn>Device Key</dfn>
+: <dfn>Secondary Key</dfn>
+:: A [=hardware-bound device key pair=], also known as a [=device-bound key=], is a [=[RP]=]-specific and [=user credential=]-specific public key pair created upon a [=[RP]=]'s request via the [=devicePubKey=] [=WebAuthn extension=].
+    The [=[RP]=] may supply this extension during both [=registration=] and [=authentication=].
+    The [=authenticator=] that a [=hardware-bound device key pair=] is created upon guarantees that the [=device private key=] is securely stored in hardware, i.e., it is unextractable. The [=device public key=] is returned only to the [=[RP]=] that created the [=hardware-bound device key pair=]. See [[#sctn-device-publickey-extension]].
```

Comment:
Is it worth calling out that this is for authenticators where the [=credential public key=] might _not_ be hardware-bound? I think in the minds of readers for some time they're probably coming to WebAuthn with that assumption and might be unmoored by this description without that context.

Reply:
Yes, good catch, I've submitted #1665 re explicitly defining the "synced | hardware-bound" credential notions.